.github/actions/setup/action.yml

```diff
@@ -26,5 +26,5 @@ runs:
       shell: bash
 
     - name: pnpm build
-      run: pnpm nx affected --base=$NX_BASE --head=$NX_HEAD --target=build
+      run: pnpm nx affected --base=$NX_BASE --head=$NX_HEAD --target=build --exclude=api-plugin
```

Comment:
Theres no `build` command for `api-plugin` so i don't think we need to worry about excluding `api-plugin` i think it would be good to make sure that all the linting atleast works for this. Since this is using nx affected it will only run this on changes so it won't build every run